pkg/kubelet/kubelet_node_status.go

```diff
@@ -545,6 +545,7 @@ func (kl *Kubelet) setNodeAddress(node *v1.Node) error {
 func (kl *Kubelet) setNodeStatusMachineInfo(node *v1.Node) {
 	// Note: avoid blindly overwriting the capacity in case opaque
 	//       resources are being advertised.
+	//glog.Info("Error getting machine info: %v", err)
```

Comment:
nit: is this a error info?

Reply:
Thanks for pointing out. Left by mistake.